#>>>>>>>>>Smartcar Lab
laser_frame: "velodyne"                                 # smartcar_lab
imu_topic: "/imu"               # smartcar_lab
lidar_topic: "/carla/ego_vehicle/lidar/front/point_cloud"                           # smartcar_lab

#>>>>>>>>>Ese
# laser_frame: "laser"                                  # ese
# imu_topic: "/imu/data"                 # ese
# lidar_topic:  "/lslidar_point_cloud"                    # ese

#>>>>>>>>>Switch
use_imu: false
use_odom: false
imu_upside_down: false

visualize: true
ndt_method_type: 2  # 0:pcl 1:cpu 2:gpu 3:omp 4:gpu_ptr

is_publish_map_full: false
is_publish_map_filterd_ground: true
is_publish_map_for_costmap: true
is_publish_map_for_vectormap: true
#>>>>>>>>>common params
base_frame: "base_link"
odom_topic: "/odom_raw"
incremental_voxel_update: "true"


# ndt params
ndt_resolution: 1.0
ndt_step_size: 0.1
ndt_trans_eps: 0.01
ndt_max_iter: 20
voxel_leaf_size: 1.0   # matching map

# scan-points filter params
min_scan_range: 2.0
max_scan_range: 50.0


# global map-set params
min_add_scan_shift: 2.0  # add to voxeled_local_map
min_update_target_map: 2.0  # add to global_map

global_voxel_leafsize: 2.0   # (self defined) refiltered map-pub voxel leaf-size

extract_length: 60  # extract target_map from voxeled_local_map
extract_width: 50

tf_timeout: 1.0
